src/components/Match/Draft/Draft.jsx

```diff
@@ -144,8 +144,13 @@ const orderOne = [1, 3, 5, 7, 10, 12, 14, 16, 18, 20, 21];
 const orderTwo = [2, 4, 6, 8, 9, 11, 13, 15, 17, 19, 22];
 const picks = [7, 8, 9, 10, 15, 16, 17, 18, 21, 22];
 
-const Draft = ({ radiantTeam, direTeam, draft }) => {
-  const firstIsTeamTwo = draft[0].active_team === 2;
+const Draft = ({
+  radiantTeam = {},
+  direTeam = {},
+  draft = [],
+}) => {
+  // if there is no draft data there is no meaning to firstIsTeamTwo
+  const firstIsTeamTwo = draft[0] && draft[0].active_team === 2;
```

Comment:
check draft as well just in case it's null (I think default [] does not replace null, only undefined)